typescript-data-model/order-history.ts

```diff
@@ -0,0 +1,108 @@
+interface order {
+  orderDate: string;
+  productName: string;
+  author?: string;
+  returnWindow: string;
+  total: number;
+  beforeTax: number;
+  shipTo: string;
+  orderNum: string;
+  deliveryDate: string;
+  delivered: boolean;
+  directDelivery?: boolean;
+  getProductSupport?: boolean;
+  writeAProductReview?: string;
+  buyAgain?: boolean;
+  viewItem?: boolean;
+}
+const orderHistory: order = [
```

Comment:
This would technically be of type `order[]`. `order` is just an object, but we are storing an array of objects. This was probably the error you were getting when committing!